app/lib/project/MetapolatorProject.js

```diff
@@ -20,6 +20,7 @@ define([
   , 'yaml'
   , 'io/zipUtil'
   , 'io/InMemory'
+  , 'filesaver'
```

Comment:
This is important even if you are not going to use it for whatever odd reason, because the next module required hereafter will need to name it's argument also, and not using the right slot causes confusion.